src/main.h

```diff
@@ -48,10 +48,10 @@ class CValidationState;
 struct CNodeStateStats;
 
 /** Default for -blockmaxsize and -blockminsize, which control the range of sizes the mining code will create **/
-static const unsigned int DEFAULT_BLOCK_MAX_SIZE = 750000;
+static const unsigned int DEFAULT_BLOCK_MAX_SIZE = MAX_BLOCK_SIZE;
 static const unsigned int DEFAULT_BLOCK_MIN_SIZE = 0;
 /** Default for -blockprioritysize, maximum space for zero/low-fee transactions **/
-static const unsigned int DEFAULT_BLOCK_PRIORITY_SIZE = 50000;
+static const unsigned int DEFAULT_BLOCK_PRIORITY_SIZE = DEFAULT_BLOCK_MAX_SIZE / 2;
```

Comment:
My recollection of the policy discussed *verbally* in the meeting agrees with @str4d. Specifically we're *limiting the non-priority* space to 1 MB (*) and "the rest of the block" will be priority space. Furthermore priority space is being repurposed to strongly favor transactions with JoinSplits.

However, as written in https://github.com/zcash/zcash/issues/1857#issuecomment-263419605 it's not clear if 1000000 bytes is a constant or incidentally half the block size, and #1901 is clear about its interpretation.

Let's keep this PR as-is, since we intend to alter this default-policy (or perhaps even introduce a consensus rule) in the future and at that time we can revisit the exact policy.